plugins/woocommerce/src/Internal/ComingSoon/ComingSoonCacheInvalidator.php

```diff
@@ -39,5 +39,15 @@ public function invalidate_caches() {
 				)
 			);
 		}
+
+		// Invalidate the SiteGround Speed Optimizer cache.
+		if ( function_exists( '\sg_cachepress_purge_cache' ) ) {
+			\sg_cachepress_purge_cache();
+		}
+
+		// Invalidate the WP Engine cache.
+		if ( class_exists( '\wpecommon' ) && method_exists( 'wpecommon', 'purge_varnish_cache' ) ) {
+			\wpecommon::purge_varnish_cache();
+		}
```

Comment:
```suggestion
```

Suggest removing this as we couldn't confirm it works on an WPE account and don't know exactly what the behavior is.